src/localization/fi.json

```diff
@@ -167,7 +167,8 @@
 
   "exhibition": {
     "reception": "Vastaanotto",
-    "visitorVariablesManagement": "Vierailijan valintojen hallinta",
+    "visitorVariables": "Vierailijan valinnat",
+    "resetVisitorVariables": "Nollaa vierailijan valintoja",
```

Comment:
Screen is called manage visitor variables, so shouldn't this be something similar?